File: src/main/java/info/dddeurope/lab/app/domain/UserAggregate/Wall.java
package info.dddeurope.lab.app.domain.UserAggregate;

import java.util.ArrayList;

import info.dddeurope.lab.app.domain.PostAggregate.Post;

public class Wall {

    ArrayList<Post> posts;

    public Wall() {
        this.posts = new ArrayList<Post>();
    }

    public void push(Post post) {
        this.posts.add(post);
    }
    
}

File: DotNet/App/Domain/UserAggregate/Wall.cs
using System.Collections.Generic;
using App.Domain.PostAggregate;

namespace App.Domain.UserAggregate
{
    public class Wall
    {
        private readonly List<Post> _posts;

        public Post[] Posts => _posts.ToArray();

        public Wall()
        {
            _posts = new List<Post>();
        }

        public void Push(Post post)
        {
            _posts.Add(post);
        }
    }
}